pom.xml

```diff
@@ -23,11 +23,45 @@
 
   <properties>
     <revision>1.99</revision>
+    <github-api.version>1.99</github-api.version>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.138.4</jenkins.version>
+    <jenkins.version>2.150.3</jenkins.version>
     <java.level>8</java.level>
     <no-test-jar>false</no-test-jar>
   </properties>
+  
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>enforce-property</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireProperty>
+                  <property>revision</property>
+                  <regex>\Q${github-api.version}\E</regex>
+                  <regexMessage>revision (${revision}) must equal github-api.version (${github-api.version})</regexMessage>
+                </requireProperty>
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-release-plugin</artifactId>
+        <configuration>
+          <!-- we do not want to auto-increment when releasing -->
+          <!-- use instructions at https://github.com/jenkinsci/incrementals-tools#superseding-maven-releases to release -->
+          <preparationGoals>not-set-up-for-MRP</preparationGoals>
```

Comment:
No delete this stuff or you will not be able to release! I was only pointing you to the `maven-enforcer-plugin` configuration.